tests/lean/lcnfTypes.lean.expected.out

```diff
@@ -33,15 +33,16 @@ Magma.mul : Magma → lcAny → lcAny → lcAny
 weird1 : Bool → lcAny
```

Comment:
@zwarich This was the only regression from preserving parameter names from base to mono. I can adjust the pretty printing in the test to avoid the change but actually looks more readable?